### ISSUE TITLE
Reference to undefined setting when using as source generator

### ISSUE THREAD

**Issue by parkan** [mariussoutier/sbt-unpack#1]
I can't get sbt to automatically unpackJars. I'm trying

```scala
import com.mariussoutier.sbt.UnpackKeys
sourceGenerators in Compile += Def.sequential(UnpackKeys.unpackJars).taskValue
```
(readme is missing the `.taskValue` but adding TaskKeys to Tasks doesn't make sense)

and I get

```
Reference to undefined setting: 

  thing-parent/*:unpackJars from thing-parent/compile:sourceGenerators (/Users/arkadiy/thing/build.sbt:225)
```

Plugin is enabled and manual invocation works fine

**Comment by mariussoutier** [comment]
You are right about the missing `.taskValue`. You don't need `Def.sequential` for a single task by the way, `sourceGenerators in Compile += UnpackKeys.unpackJars` would be sufficient.

Other than that I can't see why it's not working. Can you share more of the build.sbt?
